sound/soc/sof/nocodec.c

```diff
@@ -17,6 +17,13 @@ static struct snd_soc_card sof_nocodec_card = {
 	.owner = THIS_MODULE
 };
 
+static struct snd_soc_dai_link_component dmic_component[] = {
+	{
+		.name = "dmic-codec",
+		.dai_name = "dmic-hifi",
+	}
+};
+
```

Comment:
use dummy component or dai.

Reply:
@plbossart im wondering how that would work though. Dont we need the dmic-codec component to test it?

Reply:
no. The dmic-codec is not necessary, just look at the code to convince yourself of it :-)